### PR TITLE
Use arg rest/spread when possible.

### PR DESCRIPTION
**Pull request by XhmikosR**
Split from #4453 

Lint needs fixing, any help welcome!

### CODE REVIEW

File: lib/__tests__/fixtures/processor-fenced-blocks.js [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 const execall = require('execall');
 
-module.exports = function() {
-	const options = arguments.length > 0 && arguments[0] !== undefined ? arguments[0] : {};
```
Comment (hudochenkov) [inline lib/__tests__/fixtures/processor-fenced-blocks.js]: I think it could be replaced with:

```js
module.exports = function(options = {}) {
```

Comment (XhmikosR) [reply lib/__tests__/fixtures/processor-fenced-blocks.js]: Agreed, feel free to push it :)

Comment (XhmikosR) [reply lib/__tests__/fixtures/processor-fenced-blocks.js]: I pushed this, only lint is left :)

**Comment by XhmikosR** [comment]
BTW we need to add this in our ESLint rules for sure. And test Node.js 8.7.0 to make sure every feature we use still works in the minimum supported version.

**Pull request merged**